example_test.go

```diff
@@ -3,7 +3,7 @@ package lumberjack_test
 import (
 	"log"
 
-	"gopkg.in/natefinch/lumberjack.v2"
+	"github.com/natefinch/lumberjack"
```

Comment:
This is incorrect.  The original import path is correct. Importing from github directly will import v1 code.